Car.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+
+#include "Car.hpp"
+
+Car::Car() {
+    std::cout << "* " << __FUNCTION__ << std::endl;
+}
+
+Car::~Car() {
+    std::cout << "* " << __FUNCTION__ << std::endl;
+}
+
+void Car::turnLeft() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnRight() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::brake() {
+    std::cout << __FUNCTION__;
+    speed_ = 0;
+    std::cout << " : stopped" << std::endl;
+}
+
+void Car::accelerateTo(int speed) {
+    std::cout << __FUNCTION__;
+    if (speed >= 0) {
+        speed_ = speed;
+        std::cout << " : " << speed << std::endl;
+    } else {
+        std::cout << " : speed must be positive" << std::endl;
+    }
```

Comment:
I suggest you to throw exception if velocity is less than 0

Reply:
Good point,  I would do so in a final version, but here I didn't want to break the program - just see the problem and let it go further.

Reply:
Exception handling added, finally!